### PR TITLE
Update readme with proper description

### LINKED ISSUE #44: [Impl] Create a proper readme file.
Let's create a proper readme file inserting the commands that need to be run and a brief description of the repo.

Here is a sample template you can refer to: [Readme Template](https://gist.github.com/martensonbj/6bf2ec2ed55f5be723415ea73c4557c4#file-readme-template-md)

### PR DESCRIPTION
**Pull request by chathuranga-jayanath-99**
Fixes: #44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+![Alt text](./src/images/image-1.png)
+![Alt text](./src/images/image-2.png)
+Once examiner logs in he/she can view the exams on current day.
+![Alt text](./src/images/image-3.png)
+Once the examiner press on an exam. Examiner will be directed to the attendance monitoring page.
+![Alt text](./src/images/image-4.png)
+Once examiner press `Start Attending` on `Attendance Monitoring` page. This window will be opened to take attendance of students.
+![Alt text](./src/images/image-5.png)
+In case, student can be verified by the index as well.
```
Comment (Malith-19) [inline README.md]: Isn't that better to put some sub headings above the screen shots and small description about that page.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+# Installation and Setup Instructions
+
+Installation:
+
+`npm install`
+
+To Run Test Suite:
+
+`npm test`
+
```
Comment (Malith-19) [inline README.md]: Test part is not there I think.

Comment (chathuranga-jayanath-99) [reply README.md]: Okay I'll remove it

File: README.md (hunk 1 above)
Comment (chathuranga-jayanath-99) [reply README.md]: Sure. I'll add.

**Review by Malith-19 — APPROVED** [review]

**Review by jananga99 — APPROVED** [review]

**Review by sanduninduwara — APPROVED** [review]

**Pull request merged**